tests/unit/components/link-to-username-test.js

```diff
@@ -0,0 +1,41 @@
+import {
+  moduleForComponent,
+  test
+} from 'ember-qunit';
+
+moduleForComponent('link-to-username', {
+  // Specify the other units that are required for this test
+  // needs: ['component:foo', 'helper:bar']
+  unit: true
+});
+
+test('role returns the IRC role of the user', function(assert) {
```

Comment:
I did a quick check with XMPP and it would be possible to impersonate mods and ops there, as you can choose any nickname you want, incl. special characters:

![screenshot from 2017-05-11 14-49-11](https://cloud.githubusercontent.com/assets/842/25949809/19e7cd5e-3659-11e7-94a7-4a964c55d7c6.png)